lib/resque/data_store.rb

```diff
@@ -84,6 +89,11 @@ def all_resque_keys
       end
     end
 
+    def server_time
+      time, _ = @redis.time
```

Comment:
I see the issue will be resolved by updating `redis-namespace`

https://github.com/resque/redis-namespace/pull/119